docs/snippets/common/main-config-framework.js.mdx

```diff
@@ -0,0 +1,14 @@
+```js
+// .storybook/main.js
+
+export default {
+  framework: {
+    // Replace react-vite with the framework you are using (e.g., react-webpack5)
+    name: '@storybook/react-vite',
```

Comment:
Per our conversation and to align with the other examples, replace `@storybook/react-vite` with the framework agnostic text. Similar to the the TS variation

Reply:
I had left this one using `react-vite` on purpose, because the options demonstrated only apply to React projects. The comment is unique to this snippet too, saying to replace `react-vite` with `react-webpack5`, to keep it React-specific.

Make sense? Bad idea?

Reply:
Thanks for the clarification on this. appreciate it